types/index.d.ts

```diff
@@ -17,7 +17,7 @@ interface Window {
     requestFileSystem(
         type: LocalFileSystem,
         size: number,
-        successCallback: (fileSystem: FileSystem) => void,
+        successCallback: (fileSystem: FileSystemCordova) => void,
```

Comment:
Generally when we prefix, we use `CDV` prefix. Prefixes aren't normally done... but they are done by convention in iOS code, so I think it might make sense to replicate that for consistency here.

`CDVFileSystem` might be a better name for the type.